faraday-multipart.gemspec

```diff
@@ -31,5 +31,5 @@ Gem::Specification.new do |spec|
 
   spec.required_ruby_version = '>= 2.4', '< 4'
 
-  spec.add_dependency 'multipart-post', '>= 1.2', '< 3'
+  spec.add_dependency 'multipart-post', '>= 2.0.0', '< 3'
```

Comment:
I manually checked that faraday-multipart no longer works with multipart-post < 2, as the argument cardinality of some methods in Parts and other classes has changed, so we safely pin >=  2 without being a breaking change.